wpimath/src/main/native/include/frc/filter/SlewRateLimiter.h

```diff
@@ -28,13 +28,27 @@ class SlewRateLimiter {
   using Rate_t = units::unit_t<Rate>;
 
   /**
-   * Creates a new SlewRateLimiter with the given rate limit and initial value.
+   * Creates a new SlewRateLimiter with the given rate limit.
    *
    * @param rateLimit The rate-of-change limit.
+   */
+  explicit SlewRateLimiter(Rate_t rateLimit)
+      : SlewRateLimiter(rateLimit, -rateLimit) {}
+
+  /**
+   * Creates a new SlewRateLimiter with the given positive and negative rate
+   * limit and initial value.
+   *
+   * @param positiveRateLimit The rate-of-change limit in the positive
+   * direction, in units per second.
+   * @param negativeRateLimit The rate-of-change limit in the negative
+   * direction, in units per second.
    * @param initialValue The initial value of the input.
    */
-  explicit SlewRateLimiter(Rate_t rateLimit, Unit_t initialValue = Unit_t{0})
-      : m_rateLimit{rateLimit},
+  explicit SlewRateLimiter(Rate_t positiveRateLimit, Rate_t negativeRateLimit,
```

Comment:
Since the constructor has more than one non-defaulted argument, it doesn't need to be marked explicit.